pkg/apis/crds/karpenter.sh_nodepools.yaml

```diff
@@ -78,7 +81,18 @@ spec:
                       node upgrade, memory leak protection, and disruption testing.
                     pattern: ^(([0-9]+(s|m|h))+)|(Never)$
                     type: string
+                    x-kubernetes-validations:
+                    - message: expireAfter cannot be negative
```

Comment:
This is already enforced by the regex so I don't think that we need this anymore